### PR TITLE
Add email template for self sign up succesful completion.

### PR DESCRIPTION
**Pull request by denuwanthi**
### Proposed changes in this pull request

Resolves https://github.com/wso2/product-is/issues/9594

This PR will introduce new email template for self sign up email succesful confirmation

selfsignupsuccess

### CODE REVIEW

**Review by IsuraD — APPROVED** [review]

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 1]
```diff
@@ -2024,4 +2024,60 @@
...
+                <tr>
+                    <td style="text-align: left; padding: 0px 50px 20px 50px;" valign="top">
+                        <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #666; text-align: left; padding-bottom: 3%;">
+                            Hi {{user.claim.givenname}},
+                        </p>
+                        <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #666; text-align: left; padding-bottom: 3%;">
+                            Please note that your account has been successfully activated. <br>
+                            Now you can use services in {{tenant-domain}}. <br>
+                            Account activation time: {{selfsignup-confirm-time}}
+
```
Comment (IsuraD) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml:2054]: remove line

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 2]
```diff
@@ -2024,4 +2024,60 @@
...
+                        </p>
+                    </td>
+                </tr>
+
```
Comment (IsuraD) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml:2058]: remove line

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 3]
```diff
@@ -2024,4 +2024,60 @@
...
+                    </td>
+                </tr>
+                <tr>
+                    <td style="text-align: left; padding: 0px 50px 20px 50px;" valign="top">
+                        <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #666; text-align: left; padding-bottom: 3%;">
+                            Hi {{user.claim.givenname}},
+                        </p>
+                        <p style="font-size: 18px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #666; text-align: left; padding-bottom: 3%;">
+                            Please note that your account has been successfully activated. <br>
+                            Now you can use services in {{tenant-domain}}. <br>
```
Comment (IsuraD) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml:2052]: Shall we remove this line? Now you can use services in {{tenant-domain}}

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml (hunk 1 above)
Comment (denuwanthi) [reply features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Fixed with 9b1a80212f56db0c72cd04048d38175e7effb768

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml (hunk 2 above)
Comment (denuwanthi) [reply features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml:2056]: fixed with 9b1a80212f56db0c72cd04048d38175e7effb768

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml (hunk 3 above)
Comment (denuwanthi) [reply features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: fixed with 9b1a80212f56db0c72cd04048d38175e7effb768

**Pull request merged**